module1/build tools/ustils/src/main/java/com/epam/task1/StringUtils.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.task1;
+
+import org.apache.commons.lang3.math.NumberUtils;
+
+public class StringUtils {
+
+    public static void main(String[] args) {
```

Comment:
Why do you need main method in this class?

Reply:
delete method in last commit